<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
    <meta name="google-site-verification" content="7eDjHMCU1NQK0HwCiHV-Y6E9pfOe8JkAc49dcdl3mWc" />
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>AML31 Supplementary Information</title>
    <link rel="shortcut icon" href="http://genome.wustl.edu/workspace/images/favicon.ico" type="image/x-icon" />
    <link rel="stylesheet" type="text/css" href="main.css" />
  </head>

  <body class="home">
    <div class="navbar">
      <div class="navbar-inner">
        <div class="row helix">
          <img src="tgi_logo_shadow.png" title="The McDonnell Genome Institute at Washington Unversity School of Medicine" alt="The McDonnell Genome Institute at Washington Unversity School of Medicine" style="float:left;margin-left:25px;"/>



          <div class="span8 menu">
            <div class="nav-collapse">
              <ul class="nav">
                <li>
                  <a href="index.html">Main</a>
                </li>
                <li>
                  <a href="sequence.html">Sequence Data</a>
                </li>
                <li>
                  <a href="methods.html">Methods</a>
                </li>
                <li>
                  <a href="analyses.html">Analyses</a>
                </li>
                <li>
                  <a href="tools.html">Tools</a>
                </li>
                <li>
                  <a href="citation.html">Citation</a>
                </li>
              </ul>
            </div>
          </div>          
        </div>
      </div>
    </div>
    <div class="container background">
      <div class="row">
        <div class="aml31span4 main-block publications">
          <div class="block-padding">            
            <h2 class="heading">Sequence Data</h2>
            <div class="content">
              <p>Almost 3 terabases of sequence was produced from these samples. These are archived in dbGaP under accession id <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">phs000159</a>. Direct links to individual members of the dataset are below. An inventory of submitted files is available here: <a href="AML31_DBGAP_Submission_Info.xlsx">AML31_DBGAP_Submission_Info.xlsx</a>.<p>
                <table>
                  <tr>
                    <td class="aml31tablehead">Data type</td>
                    <td class="aml31tablehead">Normal sample</td>
                    <td class="aml31tablehead">Primary tumor sample</td>
                    <td class="aml31tablehead">Relapse tumor sample</td></tr>
                  <tr class="altrow">
                    <td>Total number of DNA libraries </td>
                    <td>12 libraries <br />as 15 fractions </td>
                    <td>13 libraries <br />as 23 fractions </td>
                    <td>8 libraries <br />as 10 fractions </td></tr>
                  <tr>                  
                    <td>Illumina whole genome sequencing (WGS) *</td>
                    <td>1 library, 4 reps;<br />422 Gbp; 121x cov; 2,798,334,336 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                    <td>4 libraries; 12 reps<br />1,167 Gbp; 312x cov; 2,812,535,568 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                    <td>1 library, 2 reps<br />153 Gbp; 38x cov; 2,565,812,211 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td></tr>
                  <tr class="altrow">
                    <td>Illumina exome data (NimbleGen SeqCap EZ v3.0 exome) *</td>
                    <td>1 library, 1 reps<br />57 Gbp; 263x cov; 148,727,026 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                    <td>9 libraries<br />100 Gbp; 433x cov; 174,183,401 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                    <td>3 libraries<br />77 Gbp; 251x cov; 221,017,304 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td></tr>
                  <tr>
                    <td>NimbleGen custom capture (200k sites) sequenced with Illumina * </td>
                    <td>7 libraries, 7 reps<br /> 99 Gbp;1,130x cov; 237,811,149bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                    <td>6 libraries, 6 reps<br /> 128 Gbp; 1,500x cov; 358,872,658 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                    <td>2 libraries, 2 reps<br />29 Gbp; 280x cov; 70,503,530 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td></tr>
                  <tr class="altrow">
                    <td>NimbleGen custom capture (200k sites) sequenced with Ion Torrent </td>
                    <td>1 library<br />6.1 Gbp; 43x cov; 25,367,661 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                    <td>1 library<br />6.1 Gbp; 49x cov; 30,630,716 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                    <td>1 library<br />6.6 Gbp; 45x cov; 36,074,328 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td></tr>
                  <tr>
                    <td>IDT custom capture (AML RMG set) sequenced with Illumina</td>
                    <td>4 libraries<br />3 Gbp; 270x cov; 5,767,262 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                  <td>10 libraries<br />7 Gbp; 1209x cov; 7,305,924 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                  <td>N/A</td></tr>
                <tr class="altrow">
                  <td>IDT custom capture (145 target sites) sequenced with Illumina</td>
                  <td>4 libraries, 4 reps<br />14 Gbp; 6,994x cov; 2,801,436 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                  <td>3 libraries, 3 reps<br />9 Gbp; 6,939x cov; 2,586,556 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                  <td>3 libraries, 3 reps<br /> 8 Gbp; 9,713x cov; 2,378,313 bp breadth<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td></tr>
                <tr>
                  <td>ddPCR sequencing (15 targets)</td>
                  <td>N/A</td>
                  <td>6,109x cov (valid droplets)<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                  <td>5,619x cov (valid droplets)<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td></tr>
                <tr class="altrow">
                  <td>PCR Amplicon Ion Torrent Sequencing (11 cancer driver targets). &nbsp;Additional sample timepoints not summarized here.</td>
                  <td>6,846x cov;<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                  <td>12,299x cov;<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                  <td>13,725x cov;<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td></tr>
                <tr>
                  <td>Illumina RNA-seq</td>
                  <td>N/A</td>
                  <td>8 libraries; 542 Gbp;<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                  <td>1 library; 32 Gbp;<br /><a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000159">dbGaP accession</a></td>
                </tr>
              </table>
              
              <br>
              <p>
                Some of the above data sets involve capture prior to sequencing. The following BED files describe these capture reagents. Note that some of these bed files have two tracks that are concatenated together. One track correponds to the actual probe positions ('probes' or 'tiled regions'), the other track corresponds to the regions targeted by the design ('targets' or 'target regions'). For most analysis, you can just select the target region, or merge all the regions together. 
              </p>

              <table>
              <tr>
                  <td class="aml31tablehead">Bed File</td>
                  <td class="aml31tablehead">MD5 file</td>
                  <td class="aml31tablehead">Description</td>
              </tr>
              <tr class="altrow">
                  <td><a href="NimbleGen_v3_Exome.hg19.bed">NimbleGen_v3_Exome.hg19.bed</a></td>
                  <td><a href="NimbleGen_v3_Exome.hg19.bed.md5">NimbleGen_v3_Exome.hg19.bed.md5</a></td>
                  <td>Bed file (with a single track only) describing the NimbleGen v3 exome reagent using build hg19 coordinates</td>
              </tr>
              <tr>
                  <td><a href="AML31_200k_snvs_custom_hg19.bed">AML31_200k_snvs_custom_hg19.bed</a></td>
                  <td><a href="AML31_200k_snvs_custom_hg19.bed.md5">AML31_200k_snvs_custom_hg19.bed.md5</a></td>
                  <td>Bed file (with two tracks) describing a custom capture design for ~200k SNV sites predicted in this single patient (from a primary and relapse tumor) using build hg19 coordinates</td>
              </tr>
              <tr class="altrow">
                  <td><a href="AML_RMG_hg19.bed">AML_RMG_hg19.bed</a></td>
                  <td><a href="AML_RMG_hg19.bed.md5">AML_RMG_hg19.bed.md5</a></td>
                  <td>Bed file (with two tracks) describing a custom AML recurrently mutated gene panel design using build hg19 coordinates</td>
              </tr>
              </table>

            </div>
          </div>
        </div>
      </div>


    </div>
    <div class="container footer">
      <div class="row">
        <div class="span11 social-links" style="display: inline-block;">
          <ul>
            <li>
              <a href="http://www.facebook.com/pages/The-Genome-Institute-at-Washington-University/107175642653384?ref=ts">
                <img src="logos/facebook.png" width="32" height="32" />
              </a>
            </li>
            <li>
              <a href="http://www.flickr.com/photos/genomeinstitute/">
                <img src="logos/flickr.png" width="32" height="32" />
              </a>
            </li>
            <li>
              <a href="https://plus.google.com/108141467205273112477">
                <img src="logos/google_plus.png" width="32" height="32" />
              </a>
            </li>
            <li>
              <a href="http://www.linkedin.com/groups?gid=3113357">
                <img src="logos/linkedin.png" width="32" height="32" />
              </a>
            </li>
            <li>
              <a href="http://www.twitter.com/GenomeInstitute">
                <img src="logos/twitter.png" width="32" height="32" />
              </a>
            </li>
            <li>
              <a href="http://en.wikipedia.org/wiki/The_Genome_Institute">
                <img src="logos/wikipedia.png" width="32" height="32" />
              </a>
            </li>
            <li>
              <a href="http://www.youtube.com/user/TheGenomeinstitute">
                <img src="logos/youtube.png" width="32" height="32" />
              </a>
            </li>
            <li>
              <a href="/rss">
                <img src="logos/rss_icon.png" width="32" height="32" />
              </a>
            </li>
          </ul>
          <br />
          <br />&nbsp;
        </div>
      </div>
    </div>
    <div class="container copyright">
      <div class="row">
        <div class="span12">
          <span class="copyright">
            Copyright © 1993-2014 Washington University in St. Louis. All rights reserved.
          </span>
          <span class="wustl-logo">
            <a href="http://medschool.wustl.edu/">
              <img src="logos/wustl-med-school-logo.svg" width="260" height="37" title="Washington University School of Medicine" alt="Washington University School of Medicine" />
            </a>
          </span>
        </div>
      </div>
    </div>
  </body>
</html>
